jazz_create-serverless-service/index.js

```diff
@@ -119,8 +114,7 @@ module.exports.handler = (event, context, cb) => {
                         reject(err);
                     } else {
                         if (response.statusCode <= 299) { // handle all 2xx response codes as success
-                            var messageToBeSent = "Your service code will be available at " + config.BIT_BUCKET_URL + bitbucketName + "/browse";
-                            resolve(messageToBeSent);
+                            resolve("Successfully created your service.");
```

Comment:
Remove BIT_BUCKET_URL from config file(s)?

Reply:
In cooperated the change